backend/go/image/stablediffusion-ggml/Makefile

```diff
@@ -76,21 +94,35 @@ ifneq (,$(findstring sycl,$(BUILD_TYPE)))
 	mkdir -p build && \
 	cd build && \
 	cmake $(CMAKE_ARGS) ../../../../../sources/stablediffusion-ggml.cpp && \
-	cmake --build . --config Release"
+	cmake --build . --config Release -j$(nproc)"
```

Comment:
this could bust our CIs

Reply:
Ah, it's already in llama ccp's build now from my last pr, but I guess it should be passed down from the main makefile invocation

Reply:
OK, it appears that just passing -j to the top level Makefile I get 100% CPU usage, probably meaning the nest make process launched by cmake cooperates with the top level one. Now I feel silly.